From 5693799d106978ec4d80649c4b76116d75475f44 Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Mon, 5 Aug 2013 10:38:31 +0800
Subject: [PATCH 228/310] ENGR00273813 MX53's TARGET_CPU_VARIANT not aligned
 with bionic update

TARGET_CPU_VARIANT should be one of the directory name under
bionic/libc/arch-arm/

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 imx5x/BoardConfigCommon.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/imx5x/BoardConfigCommon.mk b/imx5x/BoardConfigCommon.mk
index 50003c6..0d849f9 100755
--- a/imx5x/BoardConfigCommon.mk
+++ b/imx5x/BoardConfigCommon.mk
@@ -9,7 +9,7 @@ TARGET_CPU_ABI2 := armeabi
 TARGET_CPU_SMP := false
 TARGET_ARCH := arm
 TARGET_ARCH_VARIANT := armv7-a-neon
-TARGET_CPU_VARIANT := cortex-a8
+TARGET_CPU_VARIANT := generic
 ARCH_ARM_HAVE_TLS_REGISTER := true
 ARCH_ARM_USE_NON_NEON_MEMCPY := false
 
-- 
1.8.0

